samples/Voting/Data/DemoService.cs

```diff
@@ -1,42 +1,37 @@
-using System;
-using System.Threading.Tasks;
-using Microsoft.Extensions.Logging;
-using Orleans;
 using VotingContract;
 
-namespace Voting.Data
+namespace Voting.Data;
+
+public partial class DemoService
 {
-    public partial class DemoService
-    {
-        private readonly IGrainFactory _grainFactory;
-        private readonly ILogger<DemoService> _logger;
+    private readonly IGrainFactory _grainFactory;
+    private readonly ILogger<DemoService> _logger;
 
-        public DemoService(IGrainFactory grainFactory, ILogger<DemoService> logger)
-        {
-            _grainFactory = grainFactory;
-            _logger = logger;
-        }
+    public DemoService(IGrainFactory grainFactory, ILogger<DemoService> logger)
+    {
+        _grainFactory = grainFactory;
+        _logger = logger;
+    }
 
-        public async Task SimulateVoters(string pollId, int numVotes)
+    public async Task SimulateVoters(string pollId, int numVotes)
+    {
+        try
         {
-            try
+            var pollGrain = _grainFactory.GetGrain<IPollGrain>(pollId);
+            var results = await pollGrain.GetCurrentResults();
+            var random = new Random();
```

Comment:
To show "good behavior" in the samples should this use `Random.Shared` instead?
I know that totally unrelated to Orleans, but to make people aware that shared-instance avoid common troubles that may arise when new Random is used wrongly (mostly for beginners).